.travis.yml

```diff
@@ -60,7 +60,7 @@ script:
   - script/validate/vendor
   - go build -i .
   - make check
-  - if [[ "$GOOS" = "linux" && "$TRAVIS_GO_VERSION" != "1.9"* ]]; then make check-protos check-api-descriptors; fi
+  - if [ "$GOOS" = "linux" ]; then make check-protos check-api-descriptors; fi
```

Comment:
Does travis run on non-Linux at all? If so, we could remove all the `if [ "$GOOS" = "linux" ];` checks